README.md

```diff
@@ -10,4 +10,5 @@ Depth First Traversals:
 - Preorder (Root, Left, Right) : 1 2 4 5 3
```

Comment:
I really think we should do DFS and BFS, those are two classical way to traverse trees.